modules/mbedtls/zephyr_init.c

```diff
@@ -94,4 +94,15 @@ static int _mbedtls_init(const struct device *device)
 	return 0;
 }
 
+#if defined(CONFIG_MBEDTLS_INIT_HEAP)
 SYS_INIT(_mbedtls_init, POST_KERNEL, 0);
+#endif
+
+/* if CONFIG_MBEDTLS_INIT_HEAP is not defined then this function
+ * should be called by the platform before any mbedtls functionality
+ * is used
+ */
+int mbedtls_heap_init(void)
+{
+	return _mbedtls_init(NULL);
```

Comment:
`_mbedtls_init()` does more than just heap initialization. It also configured debug threshold (after rebase).

Reply:
Yes indeed, sorry I overlooked that, let me fix that right now.